active/Expander/Expander.md

```diff
@@ -0,0 +1,131 @@
+An Expander control is a UI component that provides a standard interaction for pushing adjacent content while expanding and collapsing. An Expander is independent of the contents inside it, including controls. 
+
+## Is this the right control?
+An Expander is not intended for flyout-type behaviors where a UI component overlays on top of adjacent content. In other words, Expander will 'push' and not overlay or fly out.
+
+# Examples
+<!-- Use this section to explain the features of the API, showing
+example code with each description. The general format is: 
+  feature explanation,
+  example code
+  feature explanation,
+  example code
+  etc.-->
+  
+<!-- Code samples should be in C# and/or C++/WinRT -->
+
+<!-- As an example of this section, see the Examples section for the PasswordBox control 
+(https://docs.microsoft.com/windows/uwp/design/controls-and-patterns/password-box#examples). -->
+
+
+# Remarks
+<!-- Explanation and guidance that doesn't fit into the Examples section. -->
+
+<!-- APIs should only throw exceptions in exceptional conditions; basically,
+only when there's a bug in the caller, such as argument exception.  But if for some
+reason it's necessary for a caller to catch an exception from an API, call that
+out with an explanation either here or in the Examples -->
+
+# API Notes
+<!-- Option 1: Give a one or two line description of each API (type
+and member), or at least the ones that aren't obvious
+from their name.  These descriptions are what show up
+in IntelliSense. For properties, specify the default value of the property if it
+isn't the type's default (for example an int-typed property that doesn't default to zero.) -->
+
+<!-- Option 2: Put these descriptions in the below API Details section,
+with a "///" comment above the member or type. -->
+
+
+# API Details
+<!-- The exact API, in MIDL3 format (https://docs.microsoft.com/en-us/uwp/midl-3/) -->
+~~~~
+Public enum ExpandDirection 
+{ 
+Down = 0 
+Up = 1 
+} 
+ 
+public class Expander : ContentControl 
+{ 
+Expander(); 
+ 
+public object Header {get;set;} 
+public DataTemplate HeaderTemplate { get;set; } 
+public DataTemplate HeaderTemplateSelector {get;set;} 
+ 
+public static readonly DependencyProperty HeaderProperty; 
+public static readonly DependencyProperty HeaderTemplateProperty; 
+public static readonly DependencyProperty HeaderTemplateSelectorProperty {get;} 
+ 
+public bool IsExpanded { get;set} 
+public ExpandDirection ExpandDirection { get;set;} 
+ 
+protected virtual void OnExpanded(); 
+protected virtual void OnCollapsed(); 
+ 
+public event Expanded; 
+public event Collapsed; 
+ 
+public static readonly DependencyProperty IsExpandedProperty; 
+public static readonly DependencyProperty ExpandDirectionProperty; 
+} 
+~~~~
+
+
+# Appendix
+<!-- Anything else that you want to write down for posterity, but 
+that isn't necessary to understand the purpose and usage of the API.
+For example, implementation details. -->
+
+# Inputs and Accessibility
```

Comment:
Expanding/Collapsing the expander should raise [RaisePropertyChanged](https://docs.microsoft.com/en-us/uwp/api/windows.ui.xaml.automation.peers.automationpeer.raisepropertychangedevent?view=winrt-19041#Windows_UI_Xaml_Automation_Peers_AutomationPeer_RaisePropertyChangedEvent_Windows_UI_Xaml_Automation_AutomationProperty_System_Object_System_Object_) with the property changed being the [ExpandCollapseProperty](https://docs.microsoft.com/en-us/uwp/api/windows.ui.xaml.automation.expandcollapsepatternidentifiers.expandcollapsestateproperty?view=winrt-19041#Windows_UI_Xaml_Automation_ExpandCollapsePatternIdentifiers_ExpandCollapseStateProperty) property.

Reply:
Makes sense, thanks for pointing it out! 😄

Reply:
Recommend: add to description

Reply:
Added this to the UI Automation Patterns section